### PR TITLE
Migration of account_invoice_line_no_picking_name

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1: 

You can add OCA's logo for completion.

File: account_invoice_line_no_picking_name/stock_move.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            user = self.pool['res.users'].browse(
+                cr, uid, uid, context=context)
+            user_groups = [g.id for g in user.groups_id]
+            ref = self.pool['ir.model.data'].get_object_reference(
+                cr, uid, 'account_invoice_line_no_picking_name',
+                'group_not_use_picking_name_per_invoice_line'
+            )
+            if ref and len(ref) > 1 and ref[1]:
+                group_id = ref[1]
+                if group_id in user_groups:
```
Comment (guewen) [inline account_invoice_line_no_picking_name/stock_move.py]: You should use `self.user_has_groups(cr, uid, 'account_invoice_line_no_picking_name.group_not_use_picking_name_per_invoice_line', context=context` (see https://github.com/odoo/odoo/blob/8.0/openerp/models.py#L1363-L1374). It calls `ResUsers.has_groups()` which uses a cache for the result.

**Comment by tafaRU** [comment]
@guewen just applied your tip, very useful, thanks!

**Comment by eLBati** [comment]
:+1:

**Comment by tafaRU** [comment]
@guewen could you please check if PR looks good now?

Thank you in advance.

File: account_invoice_line_no_picking_name/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        res = super(StockMove, self)._get_invoice_line_vals(
+            cr, uid, move, partner, inv_type, context=context)
+        if move:
+            if self.user_has_groups(
+                    cr, uid,
+                    'account_invoice_line_no_picking_name.'
+                    'group_not_use_picking_name_per_invoice_line',
+                    context=context
+            ):
+                res['name'] = move.name
```
Comment (adrienpeiffer) [inline account_invoice_line_no_picking_name/stock_move.py:39]: This key and this value isn't already present in res ?
https://github.com/odoo/odoo/blob/8.0/addons/stock_account/stock.py#L158

**Comment by pedrobaeza** [comment]
@tafaRU, any news about this?

**Comment by tafaRU** [comment]
@adrienpeiffer, you're right so this module is superfluous since 8.0.
I close this PR then.

**Pull request closed without merging**